Kinvey/Kinvey/RequestFactory.swift

```diff
@@ -101,6 +104,7 @@ public struct Options {
      */
     public init(
         client: Client? = nil,
+        urlSession: URLSession? = nil,
```

Comment:
Just a side note, not directly related to this ticket - the constructor of `Options` probably shouldn't expose each property in the class. As the number of properties grows, it'll get too verbose. It's ok to expect developers to set each property individually after they have created an `Options` instance.